package.json

```diff
@@ -77,7 +77,7 @@
     "webpack-flush-chunks": "^1.2.3"
   },
   "scripts": {
-    "analyze": "ANALYZE=true npm run build",
+    "analyze": "ANALYZE=true ANALYZER_MODE=static npm run build",
```

Comment:
I think you should leave `ANALYZER_MODE=static` off here.  The default behavior for a developer running this locally should be to start the analysis server.  When run via moov-builds, the CI config should run `ANALYZER_MODE=static npm run analyze`.